src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return radius * radius;
+    }
+
+    @Override
+    public String draw() {
```

Comment:
you should draw in this method
```suggestion
    @Override
    public void draw() {
```